Sources/PointFree/Enterprise.swift

```diff
@@ -357,41 +357,3 @@ extension EmailAddress {
     self.rawValue.lowercased().hasSuffix("@\(domain.rawValue.lowercased())")
   }
 }
-
-private func redirectCurrentSubscribers<Z>(
-  _ middleware: @escaping M<T2<User, Z>>
-) -> M<T2<User, Z>> {
```

Comment:
we had two versions of this helper that were slightly different for enterprise/non-enterprise flows, but I was able to consolidate them into one.